src/phase0/state_transition/mod.rs

```diff
@@ -38,7 +38,7 @@ pub enum Error {
     InsufficientValidators,
 }
 
-pub fn is_active_validator(validator: Validator, epoch: Epoch) -> bool {
+pub fn is_active_validator(validator: &Validator, epoch: Epoch) -> bool {
```

Comment:
yeah, i had noticed we will want to pass most of the bigger types by ref, nice catch here